dev/jobs/prep.sh

```diff
@@ -96,30 +103,38 @@ if [[ ${PROCESS_TROPCY} = "YES" ]]; then
     fi
 
 else
-    if [[ ${ROTDIR_DUMP} = "NO" ]]; then cp "${COMIN_OBSDMP}/${RUN_local}.t${cyc}z.syndata.tcvitals.tm00" "${COMOUT_OBS}/"; fi
+    if [[ ${ROTDIR_DUMP} = "NO" ]]; then
+       cpfs "${COMINobsproc}/${RUN_local}.t${cyc}z.syndata.tcvitals.tm00" "${COMOUT_OBS}/"
+    fi
 fi
 
 
 ###############################################################
 # Generate prepbufr files from dumps or copy from OPS
-if [[ ${MAKE_PREPBUFR} = "YES" ]]; then
-    if [[ ${ROTDIR_DUMP} = "YES" ]]; then
+if [[ ${MAKE_PREPBUFR:-"YES"} = "YES" ]]; then
+    if [[ ${ROTDIR_DUMP:-"YES"} = "YES" ]]; then
```

Comment:
Why provide a default `YES` only here?

Reply:
"NO" doesn't work currently so we want it set to "YES".

Reply:
Will look at removing the `ROTDIR_DUMP` option in a separate PR.

Reply:
See issue #3693